client/client.go

```diff
@@ -242,30 +240,19 @@ func WithMaxErrorRetry(count int) ClientOption {
 var _ Client = (*client)(nil)
 
 type client struct {
-	bc    BaseClient
-	tsobc BaseClient
-	tsoStreamBuilderFactory
-	// tsoDispatcher is used to dispatch different TSO requests to
-	// the corresponding dc-location TSO channel.
-	tsoDispatcher sync.Map // Same as map[string]chan *tsoRequest
-	// dc-location -> deadline
-	tsDeadline sync.Map // Same as map[string]chan deadline
-	// dc-location -> *lastTSO
-	lastTSMap sync.Map // Same as map[string]*lastTSO
-
+	keyspaceID uint32
+	// pdsd is for pd service discovery
+	pdsd            ServiceDiscovery
+	tsoc            *tsoClient
```

Comment:
These abbreviations are uncommon, how about `discovery` and `tso` directly? Or other clearer names

Reply:
pdsd -> svcDiscovery
tsoc -> tsoClient